consensus/bor/bor_test.go

```diff
@@ -163,7 +163,7 @@ func TestEncodeSigHeaderJaipur(t *testing.T) {
 	require.Equal(t, hash, hashWithoutBaseFee)
 }
 
-func TestCalcProducerDelayVeBlop(t *testing.T) {
+func TestCalcProducerDelayRio(t *testing.T) {
 	t.Parallel()
 
 	// Test cases for VeBlop condition in CalcProducerDelay
```

Comment:
The comments and test case description still use veblop. Maybe it's okay as it helps to identify the technical aspect of the change but just pointing out.

Reply:
Thanks for pointing this out. Yes, keeping veblop in some functions was intentional. It will be easier to understand what the function actually does in certain context.